apps/web/pages/settings/security/two-factor-auth.tsx

```diff
@@ -36,7 +36,7 @@ const TwoFactorAuthView = () => {
   const isCalProvider = user?.identityProvider === "CAL";
   return (
     <>
-      <Meta title={t("2fa")} description={t("2fa_description")} />
+      <Meta title={t("2fa")} description={t("password_description")} />
```

Comment:
We actually need to fix the text for `2fa_description` here IMHO. This is 2 factor page and not password page.

Reply:
what text should i use here?

Reply:
"Set up your Two-factor authentication"